code/modules/projectiles/ammo_datums.dm

```diff
@@ -961,6 +961,28 @@
 	stamina_damage = 10
 	shrapnel_chance = 0
 
+/datum/ammo/bullet/smg/mp27
+	name = "simple submachinegun bullet"
+	damage = 50
```

Comment:
you are absolutely not giving this more damage than a rifle

Reply:
its 40 nowe